src/components/CodeEditor/CodeEditor.example.js

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Do we need this file?

Reply:
Do you mean why we need to separate the CodeEditor as a component?

Reply:
Yes. I didn't understand really well the need of this additional component so I decided to ask you and confirm. But, if it is something necessary, nevermind.

Reply:
I did it just to make a separation of concerns, in case I needed to change the code editor component dependency, I wouldn't need to change the playground itself, also I thought that was cool to let other people use it for other cases...  But it is not necessary, If you think it is better to use it outside a component, I can remove it.

Reply:
I think it is good. Thanks for the explanation.